shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -68,8 +70,8 @@ - (instancetype)initWithProject:(FlutterDartProject*)projectOrNil
     _weakFactory = std::make_unique<fml::WeakPtrFactory<FlutterViewController>>(self);
     _engine.reset([[FlutterEngine alloc] initWithName:@"io.flutter" project:projectOrNil]);
     _flutterView.reset([[FlutterView alloc] initWithDelegate:_engine opaque:self.isViewOpaque]);
-    [_engine.get() runWithEntrypoint:nil];
-    [_engine.get() setViewController:self];
+    [_engine.get() createShell:nil libraryURI:nil];
+    _engineNeedsLaunch = true;
```

Comment:
Use `BOOL` rather than `bool` and `YES` rather than true for Obj-C interfaces.

Reply:
Done